cluster-autoscaler/cloudprovider/gce/gce_cloud_provider_test.go

```diff
@@ -28,6 +28,8 @@ import (
 
 	"strings"
 
+	"fmt"
```

Comment:
same as above - there should be no empty line above fmt

Reply:
done